code/game/turfs/simulated/walls_misc.dm

```diff
@@ -1,6 +1,6 @@
 /turf/simulated/wall/cult
-	name = "wall"
-	desc = "The patterns engraved on the wall seem to shift as you try to focus on them. You feel sick."
+	name = "runed wall"
+	desc = "A wall made of a strange alloy. It's shockingly cold to the touch."
```

Comment:
My headcanon behind runed metal is that it leeches heat from whoever touches it. It's not actually very cold - it's only cold because when you touch it your hand immediately becomes freezing cold from the metal's effects.